index.js

```diff
@@ -543,147 +543,158 @@ async function html(urls, options) {
  */
 
 async function epubgen(data, output_path, options) {
```

Comment:
This diff looks huge, I'll point out the actual changes below.

Reply:
GitHub's 'hide whitespace changes' feature also works pretty well in this case btw :-)